store/tikv/kv.go

```diff
@@ -56,7 +56,6 @@ func createEtcdKV(addrs []string, tlsConfig *tls.Config) (*clientv3.Client, erro
 		Endpoints:        addrs,
 		AutoSyncInterval: 30 * time.Second,
 		DialTimeout:      5 * time.Second,
-		TLS:              tlsConfig,
```

Comment:
Is this removal of `TLS` field intentional?

@lysu  @jackysp @zimulala cc @3pointer

Reply:
I think this is an accident...